cwe_api/Dockerfile

```
@@ -4,9 +4,23 @@ WORKDIR /app
 
 COPY . .
 
-RUN pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cpu
+RUN apt-get update -y \
```

Hadolint

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)